cypress/e2e/reviewRequests.spec.ts

```diff
@@ -108,127 +157,510 @@ describe("Review Requests", () => {
       // NOTE: We define changes as being
       // 1. Adding a new page
       // 2. Editing an existing page
       // 3. Deleting an existing page
       // 4. Rename an existing page
       // 5. Moving an existing page
+
+      addUnlinkedPage(
+        `${TEST_PAGE_ADD}.md`,
+        TEST_PAGE_ADD,
+        "/permalink",
+        "blahblahblah",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      editUnlinkedPage(
+        `${TEST_PAGE_EDIT}.md`,
+        "some asdfasdfasdf content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      moveUnlinkedPage(
+        `${TEST_PAGE_MOVE}.md`,
+        "example-folder",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      deleteUnlinkedPage(`${TEST_PAGE_DELETE}.md`, E2E_EMAIL_TEST_SITE.repo)
+      renameUnlinkedPage(
+        `${TEST_PAGE_RENAME}.md`,
+        `new ${TEST_PAGE_RENAME}.md`,
+        E2E_EMAIL_TEST_SITE.repo
+      )
+      createReviewRequest(
+        MOCK_REVIEW_TITLE,
+        [E2E_EMAIL_ADMIN_2.email],
+        MOCK_REVIEW_DESCRIPTION
+      )
+      visitE2eEmailTestRepo()
+
       // Act
+      getViewReviewRequestButton().click()
+      cy.wait(`@${RR_OVERVIEW_INTERCEPTOR}`)
+
       // Assert
-    })
-    it("should be able to have the request approved as a collaborator", () => {
-      // Arrange
-      cy.setEmailSessionDefaults("Email collaborator")
-      // Act
-      // Assert
+      cy.contains(`${TEST_PAGE_ADD}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(`${TEST_PAGE_ADD}.md`).should(
+        "not.be.disabled"
+      )
+      cy.contains(`${TEST_PAGE_EDIT}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(`${TEST_PAGE_EDIT}.md`).should(
+        "not.be.disabled"
+      )
+      cy.contains(`${TEST_PAGE_MOVE}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(`${TEST_PAGE_MOVE}.md`).should(
+        "not.be.disabled"
+      )
+      cy.contains(`${TEST_PAGE_MOVE}.md`)
+        .parent()
+        .contains(TARGET_COLLECTION)
+        .should("exist")
+      cy.contains(`${TEST_PAGE_DELETE}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(`${TEST_PAGE_DELETE}.md`).should(
+        "be.disabled"
+      )
+      cy.contains(`new ${TEST_PAGE_RENAME}.md`).should("exist")
+      getReviewRequestOverviewPageEditButton(
+        `new ${TEST_PAGE_RENAME}.md`
+      ).should("not.be.disabled")
     })
   })
-  describe.skip("has approved review requests", () => {
+  describe("has approved review requests", () => {
     before(() => {
-      addAdmin(E2E_EMAIL_COLLAB.email)
-      // NOTE: Create a review request if none exists
-      listReviewRequests().then((requests) => {
-        if (!requests || !requests.length) {
-          createReviewRequest(
-            MOCK_REVIEW_TITLE,
-            [E2E_EMAIL_COLLAB.email],
-            MOCK_REVIEW_DESCRIPTION
-          ).then((id) => approveReviewRequest(id))
-        }
+      addAdmin(E2E_EMAIL_ADMIN_2.email)
+      closeReviewRequests()
```

Comment:
This is in case of permissions issues - one possible area of inconsistency is that a test suite could be run in which the RR was created by a different user (e.g. ADMIN_2), then failed with the RR still open - when rerunning the test, the close would fail because the test user is not the creator of the RR